package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "kaholo-plugin-git",
+  "description": "Kaholo plugin for integration with Git",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "npx eslint .",
+    "lint:fix": "npx eslint . --fix"
+  },
+  "author": "Kaholo Group(Amit Alon and Matan Kadosh)",
+  "license": "ISC",
+  "dependencies": {
+    "compare-versions": "^4.1.3",
```

Comment:
Please remove the `^` to make the versions sprcific, otherwise it defines a set of versions which could include a faulty one.